### PR TITLE
Add countries list

### PR DESCRIPTION
**Pull request by kristinaluxgroup**
Added the country list here because we have it in customer portal, vendor portal and now I need it for admin portal. It is used in the traveller form.

### CODE REVIEW

File: lib/countries.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    'Latvia',
+    'Lebanon',
+    'Lesotho',
+    'Liberia',
+    'Libya',
+    'Liechtenstein',
+    'Lithuania',
+    'Luxembourg',
+    'Macau SAR China',
+    'Macedonia',
```
Comment (nizhu) [inline lib/countries.js]: does anyone more familiar with eastern european politics know if this should be `North Macedonia` now?

Comment (kristinaluxgroup) [reply lib/countries.js]: not accepted yet I think

Comment (kristinaluxgroup) [reply lib/countries.js]: should be FYROM infact

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lib-regions",
-  "version": "3.5.0",
+  "version": "3.5.1",
```
Comment (nizhu) [inline package.json]: don't do any versioning in these commits. get it merged and then run `npm version patch`, it'll create a commit for it and generate the tag

Comment (kristinaluxgroup) [reply package.json]: just removed it after consulting with @eddiemcleanlux

**Pull request merged**